### DISCUSSION TITLE
How to fix this with pnpm?

### DISCUSSION THREAD

**Discussion by uigywnkiub** [heroui-inc/heroui#4415] [Help]
my stack:
react 19, nextjs 15
pnpm 9.15.1
![image](https://github.com/user-attachments/assets/49df4deb-6922-49c6-979c-89deb9b7090d)

Also I have errors like on Select due virtilization props invalid and etc..

**Comment by uigywnkiub** [comment]
@Pzx1997 @jrgarciadev

**Comment by wingkwong** [comment]
This requires upstream dependencies to support the target version. Overriding in package.json is an alternative but it may cause problems tho.

**Reply by uigywnkiub** [reply]
> This requires upstream dependencies to support the target version. Overriding in package.json is an alternative but it may cause problems tho.

I override this but it get the same problems for virtualization for example.

**Reply by wingkwong** [reply]
are you having this error: https://github.com/nextui-org/nextui/issues/4403?

**Reply by uigywnkiub** [reply]
> are you having this error: #4403?

Yes, exactly.

**Reply by uigywnkiub** [reply]
> are you having this error: #4403?

I added temp fix, isVirtualized={false}
